src/qt/walletcontroller.cpp

```diff
@@ -78,6 +78,23 @@ void WalletController::closeWallet(WalletModel* wallet_model, QWidget* parent)
     removeAndDeleteWallet(wallet_model);
 }
 
+void WalletController::closeAllWallets(QWidget* parent)
+{
+    QMessageBox::StandardButton button = QMessageBox::question(parent, tr("Close all wallets"),
```

Comment:
We want to avoid these sync calls (these create a temporary event loop) so here there's 2 options:
 - merge this and later remove all usages in the code
 - fix this one now

Not sure what is preferable.

Reply:
Just use exec and a lambda?